### PR TITLE
Add db server

### PR DESCRIPTION
**Pull request by Herover**
Had to base this branch on #119 's branch to keep the server in terraforms state, the other one should maybe be merged first.

### CODE REVIEW

File: terraform/volumes.tf [hunk 1]
```diff
@@ -12,3 +12,16 @@
 #  droplet_id = digitalocean_droplet.web.id
 #  volume_id  = digitalocean_volume.minitwit-data.id
 #}
+
+resource "digitalocean_volume" "db" {
```
Comment (pizzaluc) [inline terraform/volumes.tf:16]: just to understand, this is a pure storage node, and the other resource is the actual computing node?

Comment (Herover) [reply terraform/volumes.tf:16]: The digitalocean_volume's are just sort of virtual hard drives that can be moved around and expand in size, here's DOs own explanation of them https://docs.digitalocean.com/products/volumes/

**Comment by smilladion** [comment]
I set up the database on the new IP, and merged all my new commits into this PR. Everything should work now, however we need to figure out how to add the environment variables for the DB username and password to the web server.

**Review by smilladion — APPROVED** [review]
New changes look great!

**Pull request merged**